src_assets/common/assets/web/public/assets/locale/en.json

```
@@ -227,6 +227,8 @@
     "hevc_mode_desc": "Allows the client to request HEVC Main or HEVC Main10 video streams. HEVC is more CPU-intensive to encode, so enabling this may reduce performance when using software encoding.",
     "high_resolution_scrolling": "High Resolution Scrolling Support",
     "high_resolution_scrolling_desc": "When enabled, Sunshine will pass through high resolution scroll events from Moonlight clients. This can be useful to disable for older applications that scroll too fast with high resolution scroll events.",
+    "stream_audio" : "Stream Audio""
+    "stream_audio_desc" : "Weather to Stream Audio or Not, Disabling this can be useful for streaming headless displays as second monitors.",
```

```suggestion
    "stream_audio": "Stream Audio",
    "stream_audio_desc": "Whether to stream audio or not. Disabling this can be useful for streaming headless displays as second monitors.",
```

Could you also put the keys in alphabetical order?